packages/config/package.json

```
@@ -17,6 +17,7 @@
     },
     "homepage": "https://github.com/RedHatInsights/frontend-components/tree/master/packages/config#readme",
     "dependencies": {
+        "@redhat-cloud-services/frontend-components-config-utilities": "^1.0.0",
```

I would use `>=` in here.
```suggestion
        "@redhat-cloud-services/frontend-components-config-utilities": ">=1.0.0",
```

---

I dunno. I would assume that 2.x will have breaking changes which may mess up the end user.